File: docs/components_page/components/tooltip/tooltip.jl
using DashBootstrapComponents, DashHtmlComponents

tooltip = html_div([
    html_p([
        "I wonder what ",
        html_span(
            "floccinaucinihilipilification",
            id = "tooltip-target",
            style = Dict("textDecoration" => "underline", "cursor" => "pointer"),
        ),
        " means?",
    ]),
    dbc_tooltip(
        "Noun: rare, " * "the action or habit of estimating something as worthless.",
        target = "tooltip-target",
    ),
]);

File: components_page/components/tooltip/tooltip.py
import dash_bootstrap_components as dbc
from dash import html

tooltip = html.Div(
    [
        html.P(
            [
                "I wonder what ",
                html.Span(
                    "floccinaucinihilipilification",
                    id="tooltip-target",
                    style={"textDecoration": "underline", "cursor": "pointer"},
                ),
                " means?",
            ]
        ),
        dbc.Tooltip(
            "Noun: rare, " "the action or habit of estimating something as worthless.",
            target="tooltip-target",
        ),
    ]
)
